### ISSUE TITLE
Enhance a11y for `InfiniteScrollArea`

### ISSUE THREAD

**Issue by hirotomoyamada** [yamada-ui/yamada-ui#1245]
path: stories/components/data-display/infinite-scroll-area.stories.tsx

### Description
5 accessibility violations were found in the `InfiniteScrollArea`.

#### [withRoot](https://yamada-ui.github.io/yamada-ui/?path=/story/components-data-display-infinitescrollarea--with-root)

Expected the HTML found at $('.css-qjk8z1') to have no violations:
    <section class="ui-container css-qjk8z1">

    Received:
    Scrollable region must have keyboard access (scrollable-region-focusable)

    Fix any of the following:
      Element should have focusable content
      Element should be focusable

    You can find more information on this issue here: 
    https://dequeuniversity.com/rules/axe/4.9/scrollable-region-focusable?application=axeAPI

#### [withOverflow](https://yamada-ui.github.io/yamada-ui/?path=/story/components-data-display-infinitescrollarea--with-overflow)

Expected the HTML found at $('.ui-infinite-scroll-area') to have no violations:
    <div class="ui-infinite-scroll-area css-1eyuuom">

    Received:
    Scrollable region must have keyboard access (scrollable-region-focusable)

    Fix any of the following:
      Element should have focusable content
      Element should be focusable

    You can find more information on this issue here: 
    https://dequeuniversity.com/rules/axe/4.9/scrollable-region-focusable?application=axeAPI

#### [withOrientation](https://yamada-ui.github.io/yamada-ui/?path=/story/components-data-display-infinitescrollarea--with-orientation)

Expected the HTML found at $('.ui-infinite-scroll-area') to have no violations:
    <div class="ui-infinite-scroll-area css-1wrupwd">

    Received:
    Scrollable region must have keyboard access (scrollable-region-focusable)

    Fix any of the following:
      Element should have focusable content
      Element should be focusable

    You can find more information on this issue here: 
    https://dequeuniversity.com/rules/axe/4.9/scrollable-region-focusable?application=axeAPI

#### [isDisabled](https://yamada-ui.github.io/yamada-ui/?path=/story/components-data-display-infinitescrollarea--is-disabled)

Expected the HTML found at $('.ui-infinite-scroll-area') to have no violations:
    <div class="ui-infinite-scroll-area css-1eyuuom">

    Received:
    Scrollable region must have keyboard access (scrollable-region-focusable)

    Fix any of the following:
      Element should have focusable content
      Element should be focusable

    You can find more information on this issue here: 
    https://dequeuniversity.com/rules/axe/4.9/scrollable-region-focusable?application=axeAPI

#### [useReset](https://yamada-ui.github.io/yamada-ui/?path=/story/components-data-display-infinitescrollarea--use-reset)

Expected the HTML found at $('.ui-infinite-scroll-area') to have no violations:
    <div class="ui-infinite-scroll-area css-1eyuuom">

    Received:
    Scrollable region must have keyboard access (scrollable-region-focusable)

    Fix any of the following:
      Element should have focusable content
      Element should be focusable

    You can find more information on this issue here: 
    https://dequeuniversity.com/rules/axe/4.9/scrollable-region-focusable?application=axeAPI

### Problem Statement/Justification
A11y is essential to ensure quality when operating a package.

### Additional Information
While it's ideal to cover all jobs, if there are too many jobs to work on, please create a PR within your capacity.
Any deficiencies not covered by the PR will be automatically created as issues.
